surfinBH/_fit_evaluators/fit_7dq4Emri.py

```diff
@@ -0,0 +1,267 @@
+import numpy as np
+from surfinBH import surfinBH
+import warnings
+
+#=============================================================================
+class Fit7dq4Emri(surfinBH.SurFinBH):
+    """ A class for the NRSur7dq4EmriRemnant model presented in Boschini et al.,
+    arxiv:[TO_BE_CHANGED] (hereafter THE PAPER). 
+    
+    This model predicts the final mass mf and final spin chif, for the remnants 
+    of precessing binary black hole systems,  extending to arbitrarily large 
+    mass ratios.
+    The fits are done using Gaussian Process Regression (GPR) and also 
+    provide an error estimate along with the fit value.
+
+    This model has been trained in the parameter space:
+        NR: q <= 4, |chiA| <= 0.8, |chiB| <= 0.8
+        EMRI: 100 <= q <= 1000 , |chiA| <= 1, |chiB| <= 1
+
+    It can be evaluated at arbitrary mass ratios.
```

Comment:
A bit awkward. How about my original suggestion:
"But, the model can be evaluated at arbitrary mass ratios and spins"

Reply:
Done